sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/VectorIndexSpec.java

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Constants;
+import com.azure.cosmos.implementation.JsonSerializable;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+/**
+ * Vector Indexes spec for Azure CosmosDB service.
+ */
+public final class VectorIndexSpec {
+
+    private final JsonSerializable jsonSerializable;
+    private String type;
+
+    /**
+     * Constructor
+     *
+     * @param path the path.
+     */
+    public VectorIndexSpec(String path) {
+        this.jsonSerializable = new JsonSerializable();
+        this.setPath(path);
+    }
+
+    /**
+     * Constructor.
+     */
+    public VectorIndexSpec() {
```

Comment:
Similar to previous comments - we don't need to have 2 constructor overloads when we have the setter method.

Reply:
Done